### PR TITLE
translated Getting Started in italian

### LINKED ISSUE #12498: Italian translation of Getting Started
- [x] Configure your Code Portfolio (update)
- [x] Read Coding News on our Medium Publication

### PR DESCRIPTION
**Pull request by jlouiss**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [x] Add new translation (feature adding new translations)

#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issue: Closes #12498 

#### Description
Completed translation for getting-started.json

### CODE REVIEW

**Comment by raisedadead** [comment]
Thanks for pull request @jlouiss ! Can you get this peer reviewed with anyone? You can start by looking the chat rooms for people who would be interested.

**Comment by jlouiss** [comment]
@raisedadead I already asked on gitter, I hope people will check the PR as soon as possible

**Comment by raisedadead** [comment]
@jlouiss sure thing, we will wait for a week and post that we will merge after a sanity check if no one comes ahead for a review.

But, thanks a lot for your effort in bringing freeCodeCamp to more people. Cheers!

File: seed/challenges/00-getting-started/getting-started.json
Comment (Em-Ant) [inline seed/challenges/00-getting-started/getting-started.json]: "...una gif ~~mostrando~~ **che mostra** ..." it's better. What do you think ?

File: seed/challenges/00-getting-started/getting-started.json
Comment (Em-Ant) [inline seed/challenges/00-getting-started/getting-started.json]: i didn't like the "portfolio codice" wording too !

File: seed/challenges/00-getting-started/getting-started.json
Comment (Em-Ant) [inline seed/challenges/00-getting-started/getting-started.json]: "...~~parlando~~ **che parlano**..." same as before. In Italian the "gerundio" verbal form is not used this way.

**Review by Em-Ant — CHANGES_REQUESTED** [review]
I think there are some small issues in your translation. Anyway, thanks for your help :+1:  :smile:

**Comment by jlouiss** [comment]
I just committed the requested changes

**Comment by Em-Ant** [comment]
@jlouiss @raisedadead  LGTM :+1:

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @jlouiss @Em-Ant !

**Pull request merged**
